devices.js

```diff
@@ -1180,7 +1180,8 @@ const devices = [
 
     // TuYa
     {
-        fingerprint: [{modelID: 'TS011F', manufacturerName: '_TZ3000_rk2yzt0u'}, {modelID: 'TS0001', manufacturerName: '_TZ3000_o4cjetlm'}, {manufacturerName: '_TZ3000_o4cjetlm'}],
+        fingerprint: [{modelID: 'TS011F', manufacturerName: '_TZ3000_rk2yzt0u'},
```

Comment:
This patch looks like it might have been generated by a formatter ...

Reply:
I did this because the linter complained that the line was too long. Not sure how it got through before this PR.

Reply:
This change can be reverted, already fixed it on the master branch. (so rebase it afterwards)